tools/pre_build_script.bat

```diff
@@ -16,12 +16,16 @@
 :: limitations under the License.
 ::
 
-set TOOLS=..\..\..\tools
-set DIR=..\..\..\source\daplink
-REM  Create default version_git.h.  This is so the project 
-REM  builds if python is not installed
+@set TOOLS=..\..\..\tools
```

Comment:
I forgot you could supress individual messages with @.  This is a great idea!  It makes the build output much cleaner